packages/automerge-repo/src/DocCollection.ts

```diff
@@ -29,6 +33,15 @@ export class DocCollection extends EventEmitter<DocCollectionEvents<unknown>> {
     return handle
   }
 
+  createUnstable<T>(): DocHandle<T> {
```

Comment:
i'd rather not have a specific method for this -- i'm okay with passing in arguments somehow but because this Unstable API might not be there in other releases i don't think we should bake it in here.

Reply:
thinking more about this, i wonder if it would be better to expose a library configuration setting instead of doing this in automerge-repo, so instead of doing this you'd call something like Automerge.enableExperimental() or something... WDYT, @alexjg ?

Reply:
Noted. Will adjust for so that an option is passed in to `create`.

I had initially considered passing it in as a global option when instantiating a `Repo`,  but wanted the option of having both `stable` and `unstable` documents. The same problem might occur if there was a separate library configuration option, as it would make it more difficult to chose on a per document level?